src/tmate.py

```diff
@@ -84,18 +82,39 @@ def install_dependencies(proxy_config: typing.Optional[state.ProxyConfig] = None
         DOCKER_DAEMON_CONFIG_PATH.parent.mkdir(parents=True, exist_ok=True)
         DOCKER_DAEMON_CONFIG_PATH.touch(exist_ok=True)
         DOCKER_DAEMON_CONFIG_PATH.write_text(daemon_config, encoding="utf-8")
+
     try:
-        apt.update()
-        apt.add_package(APT_DEPENDENCIES)
+        apt.add_package("docker.io")
     except (apt.PackageNotFoundError, apt.PackageError, subprocess.CalledProcessError) as exc:
-        raise DependencySetupError("Failed to install apt packages.") from exc
+        raise DependencySetupError("Failed to apt install Docker.") from exc
     passwd.add_group("docker")
     try:
         passwd.add_user_to_group(USER, "docker")
     except ValueError as exc:
         raise DependencySetupError(f"Failed to add user {USER} to docker group.") from exc
 
 
+def install_dependencies(proxy_config: typing.Optional[state.ProxyConfig] = None) -> None:
+    """Install dependenciese required to start tmate-ssh-server container.
+
+    Args:
+        proxy_config: The proxy configuration to enable for dockerd.
+
+    Raises:
+        DependencySetupError: if there was something wrong installing the apt package
+            dependencies.
+    """
+    try:
+        apt.update()
```

Comment:
Same here. this can be combined in a single line

Reply:
Nice! Thank you!